### ISSUE TITLE
[Bug] KFC-66d930b6_6b4728.json (KFC 0910) Monitor Change Incorrect Offset

### ISSUE THREAD

**Issue by jjhhgg100123** [two-torial/sp2xpatcher#10]
Likely went unnoticed because it's not often used, but the monitor change patch incorrectly has the same offset as the power change of 3146222. I tested/checked locally and the values have the same distance as before, so you should just be able to subtract 152 from the value here, however outside of manually editing, I don't have the setup to test spice itself to verify it's not +-1.

**Comment by akitakedits** [comment]
Yeah I was just made aware of this in Discord, it's in all recent versions. Will be fixed shortly, sorry for the inconvenience!

**Comment by akitakedits** [comment]
This is now fixed, thank you for reporting!
